.changeset/twelve-coats-rhyme.md

```diff
@@ -0,0 +1,5 @@
+---
+"hardhat": patch
+---
+
+allow node task logging to be configurable
```

Comment:
Should be ready for release notes. In particular, should use past tense, and should mention the bug being fixed.

```suggestion
Fixed bug (#2352) preventing the disabling of node task logging via hardhat.config.
```

Reply:
Thanks! I couldn't remember the exact format we agreed on.